android/AndroidManifest.xml

```diff
@@ -74,7 +74,7 @@
     <activity android:name="ch.opengis.qfield.QFieldAppRaterActivity" />
 
   </application>
-  <uses-sdk android:minSdkVersion="23" android:targetSdkVersion="26" />
+  <uses-sdk android:minSdkVersion="21" android:maxSdkVersion="22"  android:targetSdkVersion="26" />
```

Comment:
Well I'm not familiar with it, so I'm just thinking. Does it make sense to have the targetSdkVersion higher than the max?

Reply:
it is a requirement by the play store. no way around it

Reply:
FYI: https://developer.android.com/distribute/best-practices/develop/target-sdk

Reply:
Easypeasy